src/cobra/io/sbml.py

```diff
@@ -598,7 +598,7 @@ def _sbml_to_model(
     if not libsbml.SyntaxChecker.isValidSBMLSId(model_id):
         LOGGER.error(f"'{model_id}' is not a valid SBML 'SId'.")
     cobra_model = Model(model_id)
-    cobra_model.name = model.getName()
+    cobra_model.name = model.getName() or None
```

Comment:
Above you marked SBML returning a `''` as model name by default as TODO but it seems like this fixes that, right?

Reply:
Yes. This does fix it. I can remove the TODO.

Reply:
Removed TODO